tests/v2/validation/nodescaling/scaling_custom_cluster_rke1_test.go

```diff
@@ -0,0 +1,99 @@
+//go:build (validation || infra.rke1 || cluster.custom || stress) && !infra.any && !infra.aks && !infra.eks && !infra.gke && !infra.rke1 && !cluster.any && !cluster.nodedriver && !sanity && !extended
+
+package nodescaling
+
+import (
+	"testing"
+
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/clusters"
+	nodepools "github.com/rancher/rancher/tests/framework/extensions/rke1/nodepools"
+	"github.com/rancher/rancher/tests/framework/extensions/scalinginput"
+	"github.com/rancher/rancher/tests/framework/pkg/config"
+	"github.com/rancher/rancher/tests/framework/pkg/session"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+)
+
+type RKE1CustomClusterNodeScalingTestSuite struct {
+	suite.Suite
+	client        *rancher.Client
+	session       *session.Session
+	scalingConfig *scalinginput.Config
+}
+
+func (s *RKE1CustomClusterNodeScalingTestSuite) TearDownSuite() {
+	s.session.Cleanup()
+}
+
+func (s *RKE1CustomClusterNodeScalingTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	s.session = testSession
+
+	s.scalingConfig = new(scalinginput.Config)
+	config.LoadConfig(scalinginput.ConfigurationFileKey, s.scalingConfig)
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(s.T(), err)
+
+	s.client = client
+}
+
+func (s *RKE1CustomClusterNodeScalingTestSuite) TestScalingRKE1CustomClusterNodes() {
+	nodeRolesEtcd := nodepools.NodeRoles{
+		Etcd:     true,
+		Quantity: 1,
+	}
+
+	nodeRolesControlPlane := nodepools.NodeRoles{
+		ControlPlane: true,
+		Quantity:     1,
+	}
```

Comment:
stacked nodes tend to break (etcd+controlplane) more often than others. Would adding this case break anything in the static tests?

Reply:
I don't believe it will, I can add it and make sure though.